### PR TITLE
database, redis, and elastic is now cleaned after PR merge

### PR DESCRIPTION
**Pull request by grossmannmartin**
| Q             | A
| ------------- | ---
|Description, reason for the PR| To prevent stale data on review server, the services (database, elastic, redis) are now cleaned after PR is merged/closed. 
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

----
:globe_with_meridians: Live Preview:
  - https://mg-clean-review.odin.shopsys.cloud
  - https://cz.mg-clean-review.odin.shopsys.cloud

### CODE REVIEW

File: project-base/app/tests/App/Functional/Component/Redis/RedisVersionsFacadeTest.php [hunk 1]
```diff
@@ -15,7 +15,9 @@ public function testCleanOldCache(): void
         $oldVersionKey = '20190308130158.test';
         $notVersionKey = '2019030813015.test';
 
-        $redisClient = self::getContainer()->get('snc_redis.test');
+        /** @var \Redis $redisClient */
+        $redisClient = self::getContainer()->get('snc_redis.global');
```
Comment (TomasLudvik) [inline project-base/app/tests/App/Functional/Component/Redis/RedisVersionsFacadeTest.php:19]: Why is this change necessary?

**Review by TomasLudvik — CHANGES_REQUESTED** [review]

File: project-base/app/tests/App/Functional/Component/Redis/RedisVersionsFacadeTest.php (hunk 1 above)
Comment (grossmannmartin) [reply project-base/app/tests/App/Functional/Component/Redis/RedisVersionsFacadeTest.php:19]: The test for redis version facade was not testing properly. 
The old versions were not deleted, yet the test was still green. With use of the same redis client as is used to delete old versions, we can be sure the deleting old versions work properly

**Review by TomasLudvik — APPROVED** [review]
